diff --git a/meson.build b/meson.build
index 2bba337..a6f5339 100644
--- a/meson.build
+++ b/meson.build
@@ -180,8 +180,9 @@ message('OpenVPN 3 Linux service binary directory: ' + get_option('prefix') / li
 
 #
 #  D-Bus configuration
-dbus_policy_dir = dep_dbus.get_variable('datadir') / 'dbus-1' / 'system.d'
-dbus_service_dir = dep_dbus.get_variable('system_bus_services_dir')
+# NOTE: we don't need these inside flatpak build env. setting them to `/tmp`
+dbus_policy_dir = '/tmp'
+dbus_service_dir = '/tmp'
 dbus_config = {
     'OPENVPN_USERNAME': get_option('openvpn_username'),
     'LIBEXEC_PATH': get_option('prefix') / libexec_dir,
@@ -307,9 +308,9 @@ subdir('src/dbus')
 subdir('src/selinux')
 subdir('src/policy/polkit')
 
-if dep_systemd.found()
-    subdir('distro/systemd')
-endif
+#if dep_systemd.found()
+#    subdir('distro/systemd')
+#endif
 
 if get_option('bash-completion').enabled()
    subdir('src/shell/bash-completion')