From e9ed0883fe9b8f6b8c888e27fc312fcd0fbe40c8 Mon Sep 17 00:00:00 2001
From: Jagadeesh Kotra <jagadeesh@stdin.top>
Date: Tue, 12 Jul 2022 14:59:51 +0530
Subject: [PATCH] openvpn3 flatpak

---
 src/python/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/python/Makefile.am b/src/python/Makefile.am
index 1c5ca9d..3d63643 100644
--- a/src/python/Makefile.am
+++ b/src/python/Makefile.am
@@ -34,7 +34,7 @@ noinst_PROGRAMS = \
 
 BUILT_SOURCES = openvpn3/constants.py
 
-pkgopenvpn3dir = $(pythondir)/openvpn3
+pkgopenvpn3dir = $(prefix)/lib/python$(PYTHON_VERSION)/site-packages/openvpn3
 pkgopenvpn3_PYTHON = \
 	openvpn3/__init__.py     \
 	openvpn3/ConfigParser.py \
-- 
2.37.0

